### ISSUE TITLE
Possibility of adding an "afterInit" event to the Neo input JavaScript?

### ISSUE THREAD

**Issue by timmyomahony** [spicywebau/craft-neo#412]
I'm working on adding Neo support to our [Craft Matrix Field Preview](https://github.com/weareferal/craft-matrix-field-preview) plugin.

In order to be able to render previews and optional preview modal, we need to know when the matrix field or neo input field has finished loading on the frontend.

With matrix fields, Craft have a specific `afterInit` event here:

https://github.com/craftcms/cms/blob/e851b77eac7b5783b750bb17f18c3ab598c40771/src/web/assets/matrix/src/MatrixInput.js#L128

This means we can do something like this from our plugin to know when to get started:

```
if (typeof Craft.MatrixInput !== "undefined") {
  Garnish.on(
    Craft.MatrixInput,
    "afterInit",
    {},
    function (ev) {
      // Add our functionality
    }.bind(this)
  );
}
```

Would it be possible to add something similar to the Neo Input field here:

https://github.com/spicywebau/craft-neo/blob/master/client/src/input/Input.js#L37

so that we could do something similar?

**Comment by ttempleton** [comment]
The `afterInit` event has now been included in Neo 2.8.14.  Let us know if there's anything else we can do to allow Neo support in Matrix Field Preview!

**Comment by timmyomahony** [comment]
This is great, thanks so much.
